docs/graphql-api/queries/index.mdx

```diff
@@ -19,11 +19,31 @@ keywords:
 seoFrontMatterUpdated: true
 ---
 
-# GraphQL Queries
+# Basics of Queries
 
-Queries are used to read data from the database. Hasura provides a powerful GraphQL API to query your data.
+## Introduction
 
-You can use the GraphiQL interface in the Hasura Console to explore your API and help you to write queries.
+GraphQL queries are used to **read data** by interacting with [models](/supergraph-modeling/models.mdx) in your Hasura
+DDN project.
+
+Queries can be used to return multiple records while specifying exactly which fields to include in the response. You can
+also query for unique records with the same level of control, retrieving a single result that matches your conditions.
```

Comment:
Are we meaning here `...ById` as the unique record query and the "single result that matches your conditions" as a `where:` filter query? Here they sound like the same, can we call them out as separate?